pytorch_lightning/trainer/trainer.py

```diff
@@ -1390,25 +1392,45 @@ def __set_ckpt_path(self, ckpt_path: Optional[str], model_provided: bool, model_
         if ft_checkpoints:
             ft_ckpt_path = ft_checkpoints[0].ckpt_path
             fs = get_filesystem(ft_ckpt_path)
-            if fs.exists(ft_ckpt_path):
-                return ft_ckpt_path
+            if not fs.exists(ft_ckpt_path):
+                ft_ckpt_path = None
+        else:
+            ft_ckpt_path = None
+
+        fn = self.state.fn.value
+
+        if ckpt_path is None and ft_ckpt_path is not None and self.state.fn == TrainerFn.FITTING:
+            ckpt_path = "last"
+            rank_zero_warn(
+                f"`.{fn}(ckpt_path=None)` was called without a model."
+                " Because fault tolerance is enabled, the last model of the previous `fit` call will be used."
+                f" You can pass `{fn}(ckpt_path='best')` to use the best model or"
+                f" `{fn}(ckpt_path='last')` to use the last model."
+                " If you pass a value, this warning will be silenced."
+            )
 
         if model_provided and ckpt_path is None:
             # use passed model to function without loading weights
             return
 
-        fn = self.state.fn.value
-
         if model_connected and ckpt_path is None:
-            rank_zero_warn(
+            full_msg = (
```

Comment:
can we clean this up a little? either we use f-string formatting only, or we do the string.format but mixing both is a bit harder to read and I believe it's not necessary.

Reply:
Done